### PR TITLE
feature/UI-1570 update tech stack

### PR DESCRIPTION
**Pull request by pklawikowski-schibsted**
Update grade
Update kotlin
Update libs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-# Schibsted account Android SDK
+# m10s Identity Android SDK
```
Comment (wojtekk) [inline README.md:1]:
```suggestion
# Schibsted Account Android SDK for Marketplaces
```

File: webflows/src/main/java/com/m10s/identity/webflows/tracking/M10sIdentityTrackingEvent.kt [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package com.m10s.identity.webflows.tracking
+
+import com.m10s.identity.webflows.BuildConfig
+
+const val PACKAGE_NAME = "identity-sdk-android-web"
+const val PROVIDER_COMPONENT = "m10s-identity"
```
Comment (wojtekk) [inline webflows/src/main/java/com/m10s/identity/webflows/tracking/M10sIdentityTrackingEvent.kt:6]:
```suggestion
const val PROVIDER_COMPONENT = "schibsted-account"
```

for now but might be changed later.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 
-![Build Status](https://github.com/schibsted/account-sdk-android-web/actions/workflows/ci.yaml/badge.svg)
+![Build Status](https://github.com/schibsted/m10s-identity-sdk-android-web/actions/workflows/ci.yaml/badge.svg)
 ![GitHub release (latest by date)](https://img.shields.io/github/v/release/schibsted/account-sdk-android-web)
 ![Platform](https://img.shields.io/badge/Platform-Android%2021%2B-orange.svg?style=flat)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://github.com/schibsted/account-sdk-android-web/blob/master/LICENSE)
 
-New implementation of the Schibsted account Android SDK using the web flows via
+New implementation of the m10s Identity Android SDK using the web flows via
```
Comment (wojtekk) [inline README.md:8]:
```suggestion
New implementation of the Schibsted Accoung Android SDK for Martkeplaces using the web flows via
```

File: README.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
-  found [here](https://schibsted.github.io/account-sdk-android-web/).
+  found [here](https://schibsted.github.io/m10s-identity-sdk-android-web/).
 * An example implementation of the SDK can be found
-  [here](https://github.com/schibsted/account-sdk-android-web/tree/master/app/src/main/java/com/schibsted/account/example).
+  [here](https://github.com/schibsted/m10s-identity-sdk-android-web/tree/master/app/src/main/java/com/m10s/identity/example).
 
 ## Getting started
 
-To implement login with Schibsted account in your app, please first have a look at our
+To implement login with m10s Identity in your app, please first have a look at our
```
Comment (wojtekk) [inline README.md:18]:
```suggestion
To implement login with Schibsted Account for Marketplaces in your app, please first have a look at our
```

**Review by wojtekk — CHANGES_REQUESTED** [review]
Awesome to see these changes!!!

I've left a few small suggesions

**Review by MarcinGie — APPROVED** [review]

**Pull request closed without merging**